src/palace/manager/feed/acquisition.py

```diff
@@ -558,15 +557,13 @@ def single_entry_loans_feed(
         if not annotator:
             annotator = LibraryLoanAndHoldAnnotator(circulation, None, library)
 
-        log = logging.getLogger(cls.__name__)
-
```

Comment:
The logging changes in this file are only tangentially related. I just cleaned up some of the logging here to use `LoggerMixin` while I was at it, so its easier to see where these messages are coming from in the future.